apps/web/src/components/forms/password.tsx

```diff
@@ -20,9 +21,9 @@ import { FormErrorMessage } from '../form/form-error-message';
 
 export const ZPasswordFormSchema = z
   .object({
-    currentPassword: z.string().min(6).max(72),
```

Comment:
Just to check, min(6) overwrites the min(8) in `ZPasswordSchema` right?

Reply:
Yeah I just wanted make sure that it actually overwrites it

Reply:
It does, as the preference is given to the min(6) in this case

Reply:
I think you should create a ZCurrentPasswordSchema which is `z.string().min(6).max(72)` and use that wherever a user has to put in a current password.

If this changes goes through as it currently is, we'd be locking some people out of their accounts

Reply:
Updated it to have `ZCurrentPasswordSchema `